modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -29,20 +29,21 @@ endif::[]
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Connection Timeout a| Number |  |  +++50000+++ | 
-| Socket Timeout a| Number |  |  +++50000+++ | 
-| Access Key a| String |  |  | x
-| Secret Key a| String |  |  | x
-| Try Default AWSCredentials Provider Chain a| Boolean |  |  +++false+++ | 
-| Max Connections a| Number |  |  | 
-| Region Endpoint a| String |  |  +++US_EAST_1+++ | 
+| S3 Compatible Storage URL a| String |  +++The URL to connect to in case an S3 compatible storage is used. If blank, the default AWS S3 one will be used.+++ |  | 
+| Connection Timeout a| Number |  +++The amount of time to wait (in milliseconds) when initially establishing a connection before giving up and timing out. A value of 0 means infinity, and is not recommended.+++ |  +++50000+++ | 
```

Comment:
"...before the connector gives up and times out." A value of 0 means infinity and is not recommended.

Reply:
Updated